lib/issuer.js

```diff
@@ -30,7 +35,9 @@ exports.issue = async ({credential, profileAgentRecord}) => {
   // if revocation list config option is set, create a VC status writer
   let credentialStatusWriter;
   if(useRevocationList2020) {
-    assertRevocationList2020Context({credential});
+    // FIXME: remove this and just assert it instead
```

Comment:
Is there any reason for us not to just fix this now?

Reply:
Removed and fixed here 338a8f3